src/cljs/main/broadfcui/common.cljs

```diff
@@ -260,27 +260,31 @@
        (this :-render-dropdown)
        (.foundation (js/$ (:dropdown-element @locals)))))
    :component-will-receive-props
-   (fn [{:keys [this locals]}]
+   (fn [{:keys [this]}]
      (this :-render-dropdown))
    :component-will-unmount
    (fn [{:keys [locals]}]
      (react/unmount-component-at-node (:dropdown-container @locals))
      (.remove (:dropdown-container @locals)))
    :-render-dropdown
-   (fn [{:keys [this props state after-update locals]}]
-     (let [{:keys [contents]} props
+   (fn [{:keys [this props state locals]}]
+     (let [{:keys [contents dropdown-class]} props
            {:keys [dropdown-container dropdown-id]} @locals]
        (react/render
         (react/create-element
          ;; empty string makes react attach a property with no value
-         [:div {:className "dropdown-pane" :id dropdown-id :data-dropdown ""
+         [:div {:className (str "dropdown-pane " dropdown-class) :id dropdown-id :data-dropdown ""
                 :ref (this :-create-dropdown-ref-handler)
-                :style {:whiteSpace "normal"}}
+                :style (merge
+                        {:whiteSpace "normal"}
+                        (:style-override props)
```

Comment:
Keep the `:style` convention we use elsewhere unless you can argue this is better and we should change it everywhere.